build.gradle

```diff
@@ -31,7 +31,11 @@ ext {
 repositories {
     mavenCentral()
     maven {
-        url = uri('https://jfrog.sandbox.fynarfin.io/artifactory/fyn-libs-snapshot')
+        url = uri('https://fynarfinhub.jfrog.io/artifactory/ph-ee-repo-local')
+        credentials {
+            username = "maven-repo-user"
+            password = "123QWEasd"
+        }
```

Comment:
These changes have been done as jfrog is not working rn.

Reply:
okay, remove this part once jfrog issue is fixed.

Reply:
Okay